src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.ts

```diff
@@ -613,6 +608,89 @@ export class InstructorCourseEditPageComponent implements OnInit {
       editedInstructor.privileges.sessionLevel = newSessionLevelPrivileges;
     }
 
+    if (instr.controls.role.value === 'Custom') {
```

Comment:
Are these change intended?

Reply:
The changes are made here: https://github.com/TEAMMATES/teammates/pull/9437/commits/4111125b924f147bb698bc44ff525f2ff4606c3a Not sure why it is showing it as a change...

Reply:
You need to remove irrelevant changes.

Reply:
I'll rebase my changes onto master to remove them then.